flink-core/src/test/java/org/apache/flink/api/common/eventtime/WatermarkOutputMultiplexerTest.java

```diff
@@ -58,29 +53,29 @@ public void singleImmediateIdleness() {
 
         watermarkOutput.markIdle();
 
-        assertThat(underlyingWatermarkOutput.lastWatermark(), nullValue());
-        assertThat(underlyingWatermarkOutput.isIdle(), is(true));
+        assertThat(underlyingWatermarkOutput.lastWatermark()).isNull();
+        assertThat(underlyingWatermarkOutput.isIdle()).isEqualTo(true);
```

Comment:
```suggestion
        assertThat(underlyingWatermarkOutput.isIdle()).isTrue();
```

There are some similar code in this class need to be improved as well.